docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -93,16 +116,11 @@ Preparing For a Release
 
 #. Create a clean, local Flocker release branch with no modifications:
 
-   .. note::
-
-      For a maintenance release, replace ``origin/master`` below with ``origin/flocker-${BASE_VERSION}``,
```

Comment:
This has been moved but also changed. I think that it should be `origin/release/flocker-${BASE_VERSION}` which the new process above should do.